urdfenvs/sensors/lidar.py

```diff
@@ -16,12 +20,21 @@ def __init__(self, linkId, nbRays=10, rayLength=10.0):
         self._relPositions = np.zeros(2*nbRays)
 
     def getOSpaceSize(self):
+        """
+        Getter for the dimension of the observation space.
+        """
         return self._nbRays * 2
 
     def getObservationSpace(self):
-        return gym.spaces.Box(-10, 10, shape=(self.getOSpaceSize(), ), dtype=np.float64)
```

Comment:
@maxspahn Do you agree with this edit? I thought the observation space should be larger then the hard coded 10 if the raylength is larger (or smaller) then 10.0

Reply:
Nice finding! Indeed, this should not be hardcoded. I think we can accept it in this pull request, despite being slightly off the main purpose of this PR. 
Aren't the values always positive? Can we set `0` as lower limit then?